README.md

```diff
@@ -50,3 +50,6 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+
+Make sure you create a trello board and get the key and token for it.
```

Comment:
It would be good to provide a little more support than this - we don't mention where to create the board (Trello) or what environment variables need to be set (`KAT_BOARD_ID` etc.). It would be good to also keep the environment variable keys up to date in the `.env.template` file (but we should make sure we don't add any values that are secrets!)